target/scripts/startup/setup-stack.sh

```diff
@@ -1068,27 +1088,31 @@ function _setup_mail_summary
 
 function _setup_logwatch
 {
-  _notify 'inf' "Enable logwatch reports with recipient ${LOGWATCH_RECIPIENT}"
-
   echo 'LogFile = /var/log/mail/freshclam.log' >>/etc/logwatch/conf/logfiles/clam-update.conf
-
-  echo "MailFrom = ${LOGWATCH_SENDER}" >> /etc/logwatch/conf/logwatch.conf
+  echo "MailFrom = ${LOGWATCH_SENDER}" >>/etc/logwatch/conf/logwatch.conf
 
   case "${LOGWATCH_INTERVAL}" in
-    'daily' )
-      _notify 'inf' "Creating daily cron job for logwatch reports"
-      echo "#! /bin/bash" > /etc/cron.daily/logwatch
-      echo "/usr/sbin/logwatch --range Yesterday --hostname ${HOSTNAME} --mailto ${LOGWATCH_RECIPIENT}" \
-        >>/etc/cron.daily/logwatch
-      chmod 744 /etc/cron.daily/logwatch
-      ;;
+    ( 'daily' | 'weekly' )
+      _notify 'inf' "Enable logwatch reports with recipient ${LOGWATCH_RECIPIENT}"
+      _notify 'inf' "Creating ${LOGWATCH_INTERVAL} cron job for logwatch reports"
 
-    'weekly' )
-      _notify 'inf' "Creating weekly cron job for logwatch reports"
-      echo "#! /bin/bash" > /etc/cron.weekly/logwatch
-      echo "/usr/sbin/logwatch --range 'between -7 days and -1 days' --hostname ${HOSTNAME} --mailto ${LOGWATCH_RECIPIENT}" \
-        >>/etc/cron.weekly/logwatch
-      chmod 744 /etc/cron.weekly/logwatch
+      local LOGWATCH_FILE INTERVAL
+
+      LOGWATCH_FILE="/etc/cron.${LOGWATCH_INTERVAL}/logwatch"
+      INTERVAL='--range Yesterday'
+
+      if [[ ${LOGWATCH_INTERVAL} == 'weekly' ]]
+      then
+        INTERVAL="--range 'between -7 days and -1 days'"
+      fi
+
+      cat >"${LOGWATCH_FILE}" << EOM
+#! /bin/bash
+
+/usr/sbin/logwatch ${INTERVAL} --hostname ${HOSTNAME} --mailto ${LOGWATCH_RECIPIENT}
+
+EOM
+      chmod 744 "${LOGWATCH_FILE}"
       ;;
 
     'none' )
```

Comment:
I assume you want consistency with case syntax here?

```suggestion
    ( 'none' )
```

Reply:
Yep, I want this to be in this PR as well :D I will commit this hand with the suggestion below.

Reply:
Done in d2c9aeda3725e6963105d845520d3a2e07a7aeac